.gitignore

```diff
@@ -0,0 +1,13 @@
+/.bundle/
+/.yardoc
+/Gemfile.lock
+/_yardoc/
+/coverage/
+/doc/
+/pkg/
+/spec/reports/
+/tmp/
+/vendor/
+/.idea/
+.env
+.DS_Store
```

Comment:
This is really a personal preference, BUT:
Every project will always .gitignore .DS_Store and .idea, and other OS/IDE files.
I personally would put that stuff into a global git excludes file.
But, given that everyone I know writes .gitignore files like this, perhaps it is better to leave it.